src/app/components/Search.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { ApplicationLauncher, ApplicationLauncherItem } from '@patternfly/react-core'
+import { useDocumentTitle } from '@app/utils/useDocumentTitle'
+import  fuzzysort from 'fuzzysort'
+import { SearchIcon } from '@patternfly/react-icons'
+import { Link } from 'react-router-dom'
+
+
+const Search: React.FunctionComponent<{ title: string }> = ({ title }) => {
+    const [data, setData] = React.useState([])
+    const [results, setResults] = React.useState([])
```

Comment:
In order to satisfy TypeScript, we should use the proper type for this state value which would be an empty `Fuzzysort.Results` initially. After checking the type definition I guess the following would solve the type mismatch :slightly_smiling_face: 

```suggestion
    const [results, setResults] = React.useState<Fuzzysort.Results>(Object.assign([], { total: 0 }))
```